tests/ember_debug/object-inspector-test.js

```diff
@@ -435,14 +456,11 @@ module('Ember Debug - Object Inspector', function(hooks) {
     assert.equal(message.details[1].properties[0].name, 'maritalStatus');
 
     assert.equal(message.details[2].name, 'Own Properties');
-    assert.equal(message.details[2].properties.length, 0, 'Correctly skips properties');
-
-    assert.equal(message.details[3].name, 'TestObject');
-    assert.equal(message.details[3].properties.length, 2, 'Does not duplicate properties');
-    assert.equal(message.details[3].properties[0].name, 'hasChildren');
-    assert.equal(message.details[3].properties[1].value.type, 'type-descriptor', 'Does not calculate expensive properties');
+    assert.equal(message.details[2].properties.length, 2, 'Correctly merges properties');
+    assert.equal(message.details[2].properties[0].value.isCalculated, undefined, 'Does not calculate expensive properties');
+    assert.equal(message.details[2].properties[1].name, 'hasChildren');
 
-    assert.ok(message.details[4].name !== 'MixinToSkip', 'Correctly skips properties');
+    assert.ok(message.details[3].name !== 'MixinToSkip', 'Correctly skips mixins');
```

Comment:
I'm curious why so many indices changed in the tests?

Reply:
Gotcha, thanks for the clarification! Can you explain what you mean by `parent`? To my knowledge, when you inspect an object, it should give you the info for that object, not parents and children, right?

Reply:
There are now 2 types of inheritance, the one from Ember and the one that comes from ES6.
Parent for me is now either a upper level mixin or prototype.
So, when we inspect an ES6 object I initially split it up into mixins, (where mixins are actually the prototypes of the object).
When we define a component, all properties, CPs, tracked are in the prototype of its instance. (We get the actual instance in the inspector, which doesn't have any properties set on its own (by `Object.hasOwnProperty`)). 
But it can also be that someone sets properties in the constructor. in that case it would have properties in the instance.

I merge the instance properties, with the one from its first prototype. So that both info is displayed in `Own Properties`
and more upper level prototypes are displayed as mixins.

Reply:
Gotcha, that makes sense. I think the instance and prototype are all part of the item though. I would not consider it a parent. I suppose perhaps base class might be what you mean?